app/backend/src/couchers/models.py

```diff
@@ -1047,6 +1044,13 @@ class Reference(Base):
         ),
     )
 
+    @property
+    def should_report(self):
+        """
+        If this evaluates to true, we send a report to the moderation team.
+        """
```

Comment:
The docstring of a function should not document what the caller is doing. I would have inlined this in the caller instead. But it's minor.